README.md

```diff
@@ -8,6 +8,42 @@
 
 DeFi Blockchain Light Wallet for iOS, Android & Web.
 
+## Releases
+
+DeFi Wallet has 3 releases channel and unique environment for each of those
+channel. [`app/environment.ts`](/app/environment.ts) carries the environment state for those releases channel.
+
+<details>
+<summary><b>Production</b></summary>
+
+Created by Expo Application Service and configured in [`eas.json`](/eas.json) it creates a native build
+with [`release-publish.yml`](/.github/workflows/release-publish.yml) workflow on type "published". Builds can only be
+trigger by DeFiChain engineers, they are automatically uploaded into native app store for distribution.
+
+Production environment is no debug and **MainNet** is the only network available.
```

Comment:
This line is a bit confusing. Maybe something like, In production environment, only **MainNet** is available and debugging is not available.

Reply:
```suggestion
In the production environment, only MainNet is available, and debugging is not available.
```